### DISCUSSION TITLE
Welcome to BambuLights Discussions!

### DISCUSSION THREAD

**Discussion by judge2005** [judge2005/BambuLights#1] [Announcements]
## 👋 Welcome!
Use the discussions for questions or suggestions.

**Comment by GerNavBet** [comment]
Ho, Thanks for this project, it seems very useful and well stablished. Im not really a programmer and was wondering if this can be installed on ESP32, ESP 32D or ESP32S Boards. Have tried on ESP32- S3 - Zero & C3-Zero but the web installer prompts they are not compatible. 

When using a ESP32 VROOM, the web installer just circles around in "Preparing To Install" but does not move forward. 

Thanks!

**Reply by GerNavBet** [reply]
Managed to get it running on a ESP32S, changed the cable and it worked. Ill give some feedback once all is running. Thanks again.
